addons/mail/static/src/core/common/thread_model.js

```diff
@@ -262,7 +263,16 @@ export class Thread extends Record {
     hasLoadingFailed = false;
     canPostOnReadonly;
     /** @type {luxon.DateTime} */
-    last_interest_dt = Record.attr(undefined, { type: "datetime" });
+    last_interest_dt = Record.attr(undefined, {
+        type: "datetime",
+        compute() {
+            const selfMemberLastInterestDt = this.selfMember?.last_interest_dt;
+            const lastInterestDt = this.last_interest_dt;
+            return compareDatetime(selfMemberLastInterestDt, lastInterestDt) > 0
+                ? selfMemberLastInterestDt
+                : lastInterestDt;
+        },
+    });
```

Comment:
This should not be computed, it should be the server value always.
Other parts of the code can compute other derived value if necessary.